etc/templates/error/502.html

```diff
@@ -0,0 +1,81 @@
+        </div>
+
+        <div id="login" class="standard_form inlined">
+	        <p>Site is currently down for maintenance. Please try again later.</p>
+
+        </div>
+
+        <div id="login-footer">
+	        <p>
+		        &copy; 2007-2014 University of Dundee &amp; Open Microscopy Environment<br>
```

Comment:
My apology I lied to you guys. In web we do ```build_year``` that's when my original question came from ;-) . I will update that to 2015 and that's all. Sorry for the confusion